src/domain/map/utils.ts

```diff
@@ -134,3 +138,18 @@ export function getFeatureFromHankeGeometry(geometry: HankeGeometria) {
 
   return feature;
 }
+
+/**
+ * Returns true if feature1 contains feature2.
+ *
+ * Due to a bug in Turf.js's `booleanContains` function (https://github.com/Turfjs/turf/issues/2588),
+ * this function checks that the intersection of feature1 and feature2 is equal to feature2.
+ */
+export function featureContains(
+  feature1: GeoJSONFeature<GeoJSONPolygon>,
+  feature2: GeoJSONFeature<GeoJSONPolygon>,
+): boolean {
+  const features: FeatureCollection<GeoJSONPolygon> = featureCollection([feature1, feature2]);
+  const intersected = intersect(features);
+  return (intersected && booleanEqual(intersected, feature2)) || false;
```

Comment:
When I tried this out, `booleanEqual` appeared to be sensitive to the feature's properties. The intersection will not contain properties, but `feature2` might. From the current call sites, it cannot have any properties, but it may in the future.

This case should be handled properly, or the exception should be clearly documented. In either scenario, there ought to be a test for this case.

Reply:
With concave polygons that intersect in several distinct places, the intersection might be a multipolygon. A multipolygon should never be equal to a polygon, so this should work correctly, but there could be a test that tests this case.

Reply:
I changed the equality check to do the comparison without properties and added tests for that case and also for the case that the intersection is a multipolygon.